docs/development/writingzeppelininterpreter.md

```diff
@@ -44,12 +42,11 @@ static {
 ```
 
 The name will appear later in the interpreter name option box during the interpreter configuration process.
-
 The name of the interpreter is what you later write to identify a paragraph which should be interpreted using this interpreter.
 
 ```
 %MyInterpreterName
-some interpreter spesific code...
+some interpreter specific code...
 ```
 ### Install your interpreter binary
 
```

Comment:
Once you have build....
-> Once you have built your interpreter, you can place it under the **interpreter directory** with all its dependencies.